velox/functions/lib/DateTimeFormatter.cpp

```diff
@@ -1328,7 +1329,23 @@ int32_t DateTimeFormatter::format(
               result);
           break;
         }
-        case DateTimeFormatSpecifier::WEEK_YEAR:
+        case DateTimeFormatSpecifier::WEEK_YEAR: {
+          auto year = getWeekYear(
+              static_cast<int>(calDate.year()),
+              static_cast<uint32_t>(calDate.month()),
+              static_cast<uint32_t>(calDate.day()),
+              2, // (ISO 8601) Monday = 2
```

Comment:
Wondering how we plan to support the legacy Spark behavior where this value should be 1.

Reply:
Maybe it can be obtained from `java.util.Calendar` in Java and then passed in through `QueryConf`.